avaframe/com1DFAPy/com1DFA.py

```diff
@@ -53,7 +53,7 @@
 startTime = time.time()
 # ------------------------
 # fetch input data
-demFile, relFiles, entFiles, resFile, flagEntRes = gI.getInputData(avalancheDir, cfgFull['FLAGS'], flagDev=True)
+demFile, relFiles, entFiles, resFile, flagEntRes = gI.getInputData(avalancheDir, cfgFull['FLAGS'], flagDev=False)
```

Comment:
I'd prefer to have the flagDev either:
- as feature flag in the input source directly 
or 
- in the cfg directly